.markdownlint.yaml

```diff
@@ -0,0 +1,3 @@
+MD013:
+  line_length: 100
```

Comment:
Is there a way to disable line length checks? I'm not a fan of them

Reply:
Sure, I'll disable the lint altogether then 👍